app/controllers/organizations/staff/dashboard_controller.rb

```diff
@@ -55,4 +37,16 @@ def context_authorize!
     authorize! :dashboard,
       context: {organization: Current.organization}
   end
+
```

Comment:
These two methods can be private, right?

Reply:
Yes they are indeed